### PR TITLE
Gadm preprocessing and ingestion

### PR DESCRIPTION
**Pull request by BielStela**
### General description

Small fix of gadm preprocessing and ingestion with and accidental upgrade of deps.

Detail:
- Replaces sql script with python (x100 faster) in preprocessing since we do not need any db ops to generate the intermediate csv
- Fixes the copy from csv in ingestion forcing the input columns to avoid problems with missing columns
- ~Updates h3ronpy to latest version~ it is broken for large geodataframes
- Updates base image gdal to 3.8.5 which ships with python 3.10. It was using 3.3 gdal which used py 3.8 which made lots of dependencies to be obsolete and hard to manage locally. now the requirements can be installed in any up to date environment without a causing a dependency wreck 

### Testing instructions

RUN THE `seed-data`

## Checklist before merging

- [ x] Branch name / PR includes the related Jira ticket Id.
- [ ] Tests to check core implementation / bug fix added.
- [ x] All checks in Continuous Integration workflow pass. (lol)
- [ ] Feature functionally tested by reviewer(s).
- [ ] Code reviewed by reviewer(s).
- [ x] Documentation updated (README, CHANGELOG...) (if required)

### CODE REVIEW

File: data/preprocessing/gadm/Makefile [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
-WORKDIR=data/tmp
+WORKDIR=data
 
-all: checksum
+all: upload_results
 	make clean-workdir
 
+upload_results: checksum
+	aws s3 cp $(WORKDIR)/geo_region.zip $(AWS_S3_BUCKET_URL)/processed/geo_region/
+	#rm $(WORKDIR)/geo_region.zip $(WORKDIR)/geo_region.csv
```
Comment (elpamart) [inline data/preprocessing/gadm/Makefile:11]: should we keep this comment here?

**Review by elpamart — APPROVED** [review]
I can not check the ingestion but overall looks good to me!

File: data/preprocessing/gadm/Makefile (hunk 1 above)
Comment (BielStela) [reply data/preprocessing/gadm/Makefile:11]: deleted it because we clean all the data files after finishing automatically anyway

**Comment by BielStela** [comment]
somehow messed with the requirements and still tweaking and testing locally so it works without unexpected crashes

**Review by elpamart — APPROVED** [review]

**Pull request merged**